service/framework.go

```diff
@@ -175,15 +177,53 @@ func newCRDFramework(config Config) (*framework.Framework, error) {
 		}
 	}
 
-	var crdFramework *framework.Framework
+	handlesFunc := func(obj interface{}) bool {
+		return true
+	}
```

Comment:
There are no version bundles yet and no ky package to use at all. Trying to push towards recent advances over time.